install.rdf

```diff
@@ -5,10 +5,11 @@
   <Description about="urn:mozilla:install-manifest">
     <em:id>quicktextpro@extensions.hesslow.se</em:id>
     <em:name>Quicktext Pro</em:name>
-    <em:version>0.9.11.7</em:version>
+    <em:version>0.9.12</em:version>
```

Comment:
Is it intended the the normal version (in master) has  `0.9.11.12`, and the pro will be at `0.9.12`?

Reply:
Master is also at 0.9.12

Reply:
You are right, I looked at the wrong place.